packages/loot-design/src/components/modals/ImportTransactions.js

```diff
@@ -40,7 +52,10 @@ function parseDate(str, order) {
     return v && v.length === 1 ? '0' + v : v;
   }
 
-  let parts = str.replace(/ /g, '').split(/[^0-9]/);
+  let parts = str.replace(/\s+/g, '').match(re[order]);
+  if (!parts) return null;
+
+  parts = parts.slice(1);
```

Comment:
```suggestion
  // We're only interested in the groups so slice the full matched string off of the array.
  parts = parts.slice(1);
```

It took me a second to remember why we needed to do this so I think it's worth making a note.

Reply:
No argument here. Committed.